src/main/java/seedu/commands/RemoveCommand.java

```diff
@@ -92,4 +93,5 @@ private void updateShopList(Ui ui, ShoppingList shoppingList, Item itemRemoved,
             ui.printShopUpdateMessage(addedItem, qtyInShop);
         }
     }
+    //@@author BryanElmer
```

Comment:
When closing you authorship claim, use `//@@author` without your github username after. Also, if you're going to claim all the way to the bottom, you don't have to end with `//@@author`.